src/pages/ContentfulPage.vue

```diff
@@ -89,6 +90,8 @@ const KvFrequentlyAskedQuestions = () => import('@/components/Kv/KvFrequentlyAsk
 
 const TestimonialCards = () => import('@/components/Contentful/TestimonialCards');
 
+const MultiStepCentered = () => import('@/components/Contentful/RichTextItemsCentered');
```

Comment:
Let's use `RichTextItemsCentered` for the Component Name and then also add `richTextItemsCentered` to the Content Group in Contentful.